### PR TITLE
Implement find medications command.

### LINKED ISSUE #11: Implement Find Medication Command
A feature that searches the inventory for medications by matching the given keyword.

### PR DESCRIPTION
**Pull request by yihernggggg**
This PR includes the implementation of FindCommand class which allows users to find medications using keywords. Corresponding additions have also been made to Parser class and PharmaTrackerTest.

Fixes #11

### CODE REVIEW

File: src/test/java/seedu/pharmatracker/PharmaTrackerTest.java [hunk 1]
```diff
@@ -107,4 +114,108 @@ public void inventory_multipleMedications_sizeIsCorrect() {
...
+    @Test
+    public void parser_findCommand_returnsCorrectCommandType() {
+        Command c = Parser.parse("find Paracetamol");
+        assertTrue(c instanceof FindCommand);
+    }
+
+    @Test
+    public void parser_findCommandNoKeyword_returnsNull() {
+        Command c = Parser.parse("find");
+        assertEquals(null, c);
```
Comment (karthikkathiresh) [inline src/test/java/seedu/pharmatracker/PharmaTrackerTest.java]: Well written test cases that consider edge cases too.

**Review by karthikkathiresh — APPROVED** [review]
Overall good implementation. LGTM.
